spec/models/labware_creators/plate_split_to_tube_racks_spec.rb

```diff
@@ -242,13 +250,12 @@
 
     context 'when a contingency file is not present' do
       it 'does not call the validation' do
-        expect(subject).not_to receive(:sufficient_tubes_in_racks?)
+        subject.valid?
```

Comment:
maybe check that is false?
```suggestion
        expect(subject).not_to be_valid
```